vaadin-platform-hybrid-test/frontend/views/helloworldts/hello-world-ts-view.ts

```diff
@@ -3,6 +3,7 @@ import '@vaadin/vaadin-button';
 import '@vaadin/vaadin-text-field';
 import { customElement, html } from 'lit-element';
 import { View } from '../view';
+import './hello-world-ts-view.global.css';
```

Comment:
There is no assertion about whether this is working, apart from  the fact that compilation passes.
Isn't worth some code that checks that a certain element has some style applied ?

Reply:
agree, will do